formula-lint/src/test/java/com/instacart/formula/lint/WrongFormulaUsageDetectorTest.kt

```diff
@@ -95,6 +97,7 @@ class WrongFormulaUsageDetectorTest {
         return lint()
             .files(*array)
             .issues(*issues)
+            .skipTestModes(TestMode.SUPPRESSIBLE)
```

Comment:
This seems to be needed with the latest lint version, otherwise these tests will fail.